aquadoggo/src/libp2p/config.rs

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use std::path::PathBuf;
+
+use anyhow::Result;
+use libp2p::identity::Keypair;
+use libp2p::swarm::ConnectionLimits;
+use libp2p::Multiaddr;
+use log::info;
+use serde::{Deserialize, Serialize};
+
+use crate::libp2p::identity::Identity;
+
+/// Keypair file name.
+const KEYPAIR_FILE_NAME: &str = "libp2p.pem";
+
+/// QUIC default transport port.
+const QUIC_PORT: u16 = 48648;
```

Comment:
This number is chosen from the list of unassigned ports (as defined by IANA).

Reply:
Maybe the key pair file should be called `secret` or `private-key`?

Reply:
Cool, I'll change to `2022` and `private-key`.